tests/benchmarks/app_kernel/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
-  benchmark.application:
+  benchmark.application.application:
```

Comment:
Many repeated sections, subsections, test names in your .yaml files. We need to decide leave them like this, or section, subsection and final test name should have different names.